test/test_autograd.py

```diff
@@ -3197,14 +3197,19 @@ def fn(*inputs):
                     f_args_variable = (self_variable,) + args_variable
                     f_args_tensor = (self_tensor,) + args_tensor
                     # could run the gradchecks again, but skip since we did it for the methods above.
+                    run_gradcheck = exclude_tensor_method(name, test_name) and not is_inplace and name not in EXCLUDE_GRADCHECK
                     run_functional_checks(self, test_name, name, fn,
-                                          False, f_args_variable, f_args_tensor)
+                                          run_gradcheck, f_args_variable, f_args_tensor)
 
                 # check for correct type of input.data and input.grad.data
                 if not is_inplace:
                     self_variable = create_input((self_size,), requires_grad=True)[0][0]
                     args_variable, kwargs_variable = create_input(args, requires_grad=False, call_kwargs=kwargs)
-                    output_variable = getattr(self_variable, name)(*args_variable, **kwargs_variable)
+                    if hasattr(self_variable, name):
+                        output_variable = getattr(self_variable, name)(*args_variable, **kwargs_variable)
+                    else:
+                        self_and_args_variable = (self_variable,) + args_variable
+                        output_variable = getattr(torch, name)(*self_and_args_variable, **kwargs_variable)
```

Comment:
I don't understand this -- do we have "inplace" functionals we are trying to test?  And why wouldn't they be tested above?

Reply:
I can't run gradcheck for in_place functions here, because I will get "a leaf Variable that requires grad has been used in an in-place operation"